CHANGELOG.md

```diff
@@ -1,2 +1,6 @@
 # Changelog
 
+## v7.0.0 24th June 2024
+
+- Library has refreshed and automatically generated on [Onfido OpenAPI Spec](https://github.com/onfido/onfido-openapi-spec) (release [v3.0.0](https://github.com/onfido/onfido-openapi-spec/releases/tag/v3.0.0))
+- Integration tests have also been implemented
```

Comment:
```suggestion
- Additional integration test coverage
```

Reply:
Actually integration tests were missing

Reply:
It's still an addition to the 0 coverage we add 😆